src/command/template/.gitignore

```diff
@@ -0,0 +1,2 @@
+.graphqlconfig
```

Comment:
oo what tool is `.graphqlconfig` for?

Reply:
That's for JetBrains IDEs, it lets me get completions and test queries manually.